src/script/components/InputBar/util/DraftStateUtil.ts

```diff
@@ -83,3 +87,19 @@ export const loadDraftState = async (
 
   return {...storageValue, messageReply, editedMessage};
 };
+
+export const getDraftTextMessageContent = (conversation: Conversation, storageValue: string) => {
+  const config = {
+    namespace: `DraftEditor-${conversation.id}`,
+    theme: {},
+    onError: console.error,
+  };
+
+  const editor = createEditor(config);
+  const editorState = editor.parseEditorState(storageValue);
+  const textContent = editorState.read(() => $getRoot().getTextContent());
+
+  const {setDraftMessage} = useMessageDraftState.getState();
+
+  setDraftMessage(conversation.id, textContent);
+};
```

Comment:
We already have saving/loading draft logic. By adding another layer (state with messages), we're kinda duplicating the logic, and we add more complexity. Can we reuse what we already have? Example of saving the draft:

```tsx
// RichTextEditor.tsx

saveDraftState(JSON.stringify(editorState.toJSON()));
```

I know it's JSON, not text, but maybe we can modify it somehow and reuse it?

Because without that, look how my info we store:

1. Draft state in local storage (json)
2. Draft state in local storage (text)
3. Draft messages external store (text, zustand)
4. Editor state (lexical)
5. ...